.gitattributes

```
@@ -4,3 +4,6 @@
 # Declare files that will always have LF line endings on checkout.
 *.sh text eol=lf
 *.py text eol=lf
+
+# YAML highlighting on GitHub webpage for CartoCSS MML files.
+*.mml linguist-language=YAML
```

```suggestion
*.mml linguist-language=YAML linguist-detectable
```
https://github.com/github/linguist/blob/master/docs/overrides.md#detectable

---

This will try to make MML files count for the language statistics on GitHub. I've tested it, but it doesn’t work. The [documentation](https://github.com/github/linguist/blob/master/docs/overrides.md#detectable) says:

> Languages that are not yet mentioned in languages.yml will not be included in the language statistics, even if you specify something like *.mycola linguist-language=MyCoolLang linguist-detectable in the .gitattributes file.

I suppose it is unlikely that MML will get support in the future. Therefore, I would leave this PR as-is for now.

---

This will try to make MML files count for the language statistics on GitHub. I've tested it, but it doesn’t work. The [documentation](https://github.com/github/linguist/blob/master/docs/overrides.md#detectable) says:

> Languages that are not yet mentioned in languages.yml will not be included in the language statistics, even if you specify something like *.mycola linguist-language=MyCoolLang linguist-detectable in the .gitattributes file.

I suppose it is unlikely that MML will get support in the future. Therefore, I would leave this PR as-is for now.

---

We should still set `linguist-detectable`. We're not setting the language to MML, we're setting it to YAML.

---

I've added a corresponding commit.